cms/templates/js/course-outline.underscore

```diff
@@ -244,6 +244,26 @@ if (is_proctored_exam) {
                         <% if (xblockInfo.get('due_date') && !course.get('self_paced')) { %>
                             <span class="status-grading-date"> <%- gettext('Due:') %> <%- xblockInfo.get('due_date') %> </span>
                         <% } %>
+                        <% if (course.get('self_paced') && course.get('is_custom_pls_active') && xblockInfo.get('due_num_weeks')) { %>
+                            <% if (xblockInfo.get('due_num_weeks') === 1) { %>
+                                <span class="status-grading-date"> <%- gettext('Due in:') %> <%- xblockInfo.get('due_num_weeks') %> week </span>
```

Comment:
the week and weeks here aren't getting translated i think.
also, you may be able to use ngettext to handle the plural case (not sure if it exists in underscore)

Reply:
ah what do you mean by not getting translated? That part of the code is needed for graded subsections -- but do you think I can put that somewhere else separately?

Reply:
Happy to chat with you about ngettext (and translations in general), but Matthew is correct about that being used here. https://docs.python.org/3/library/gettext.html#gettext.ngettext

Reply:
fixed!